src/tracker.js

```diff
@@ -58,7 +58,11 @@ export default class VideojsTracker extends nrvideo.VideoTracker {
       return this.player.mediainfo.name // Brightcove
     }
   }
-
+  
+  getLanguage() {
+    return this.player && this.player.language ? this.player.language : null;
```

Comment:
`player.language` is a function, not a property. You have to call the function to obtain the string that represents the language:

```javascript
  return this.player && this.player.language ? this.player.language() : null;
```

Reply:
Hi @asllop, I have modified the function.

`return this.player && this.player.language_ ? this.player.language_ : null;`

Player has property 'language_' which is referring to content language.

Reply:
I can't see the `language_` property in the official Video.JS docs: https://docs.videojs.com/player
Is this an undocumented property of the player object?

Reply:
Hi @asllop , I have updated method in tracker to 'language()'. 
'language_' is internal property it is using to return content language.  please re-review.